lib/email.js

```diff
@@ -20,12 +24,20 @@ bedrock.events.on('bedrock.test.configure', function() {
   require('./test.config');
 });
 
-var logger = bedrock.loggers.get('app');
-
 var api = {};
 module.exports = api;
 
+bedrock.events.on('bedrock.start', function(callback) {
+  brNotifications.register(api, function(err, results) {
```

Comment:
Should `bedrock-notifications` fire its own event for this or is this method preferred? What are the tradeoffs you see?